faros_event.sh

```diff
@@ -240,7 +243,11 @@ function parseFlags() {
                 dry_run=1
                 shift ;;
             --no_build_object)
-                no_build_object="true"
+                warn "no_build_object flag is deprecated, use skip_saving_run"
```

Comment:
printf couldn't handle when this message string has a -- in it so that is why the flags don't have that prefix.